pymess/backend/dialer/daktela.py

```diff
@@ -42,20 +46,33 @@ def _update_dialer_states(self, messages):
                 resp_json['result']['statuses']) else resp_json['result']['action']
             message_state = settings.DIALER_DAKTELA.STATES_MAPPING[resp_message_state]
             message_error = resp_json['error'] if len(resp_json['error']) else None
+            tts_processed = resp_json['result']['customFields']['ttsprocessed'][0]
 
             try:
                 self.update_message(
                     message,
                     state=message_state,
                     error=message_error,
                     extra_data=message.extra_data,
+                    is_done=resp_json['result']['action'] == '5' and tts_processed == '1',
                 )
             except Exception as ex:
                 self.update_message(message, state=DialerMessage.STATE.ERROR, error=force_text(ex))
                 # Do not re-raise caught exception. We do not know exact exception to catch so we catch them all
                 # and log them into database. Re-raise exception causes transaction rollback (lost of information about
                 # exception).
 
+    def bulk_check_dialer_status(self):
+        """
+        Method that find messages that are not in the final state and updates their states.
```

Comment:
Method that find**s** messages ...

Reply:
proč je vlastně tohle v tomto pull requestu, nevidím to tu nikde využité

Reply:
v predkovi je nejaky zakladny vseobecny filter na zaklade stavu ako je to aj v ostatnych backendoch. v daktela implementacii sa to filtruje inak. nechal by som to teda tako.

Reply:
to rozhodně ne, Když jsi přidal to is_done do obecného modelu tak to musíš aplikovat v obecném filteru

Reply:
OK